src/components/InputTodo.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { solid } from '@fortawesome/fontawesome-svg-core/import.macro';
+
+const InputTodo = ({ addTodo, value, handleChange }) => (
```

Comment:
- [ ] **Great Job On Destructing the Props 👏🏻 👏🏻**

**Please also make sure to Validate Those Props as it is considered a good Practice**